app/controllers/case_court_reports_controller.rb

```diff
@@ -1,17 +1,19 @@
 # frozen_string_literal: true
 
 class CaseCourtReportsController < ApplicationController
-  before_action :authenticate_user!
   before_action :set_casa_case, only: %i[show]
+  after_action :verify_authorized
 
   # GET /case_court_reports
   def index
+    authorize :case_court_reports
```

Comment:
In other index methods use are using `authorize CasaCase` for example. Can we keep these consistent? Or is there a subtle difference between the two?

Reply:
@littleforest I'll get that changed, nice catch!!